### PR TITLE
Exercise: Return To Electronic Life

### PR DESCRIPTION
**Pull request by artem-konovalov-personal**
Closed [#511](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/511)

### CODE REVIEW

File: 01-syntax/09-modules/return-to-electronic-life/lifeArea.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      for (x = 0; x < this.width; x++) {
+        value = this.space[x + y * this.width];
+
+        if (value !== null) {
+          f.call(context, value, new areaObject.Vector(x, y));
+        }
+      }
+    }
+  };
+  return areaObject;
```
Comment (nesterone) [inline 01-syntax/09-modules/return-to-electronic-life/lifeArea.js]: For my taste, it's easier to 

``` js
return {
      Vector: Vector,
      Grid : Grid
}
```

so I can use short names like Vector and Grid in my module's body

Comment (artem-konovalov-personal) [inline 01-syntax/09-modules/return-to-electronic-life/lifeArea.js]: Cool! Thanx a lot! I implemented your advice into my last commit

File: 01-syntax/09-modules/return-to-electronic-life/worldCreate.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    var acted = [];
+    this.grid.forEach(function (critter, vector) {
+      if (critter.act && acted.indexOf(critter) === -1) {
+        acted.push(critter);
+        this.letAct(critter, vector);
+      }
+    }, this);
+  };
+
+  worldObject.World.prototype.letAct = function (critter, vector) {
```
Comment (nesterone) [inline 01-syntax/09-modules/return-to-electronic-life/worldCreate.js]: The same here, you not need in `worldObejct` inside the body of your module

File: 01-syntax/09-modules/return-to-electronic-life/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<head>
+    <script src = "elife.js"></script>
```
Comment (nesterone) [inline 01-syntax/09-modules/return-to-electronic-life/index.html:8]: The question is : How your app works in a browser ?
I don`t see imports of other modules ...

**Comment by artem-konovalov-personal** [comment]
I changed code structure in `worldCreate.js` file and add all necessary modules to header in `index.html`

**Pull request merged**